docker-compose.yml

```diff
@@ -14,14 +14,25 @@ services:
       KAFKA_CREATE_TOPICS: "topic.OdeBsmJson:1:1,topic.FilteredOdeBsmJson:1:1,topic.OdeTimJson:1:1,topic.FilteredOdeTimJson:1:1"
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-  ppm:
+  ppm_bsm:
     build:
       context: .
-      dockerfile: Dockerfile.standalone
-    ports:
```

Comment:
Do we not need to expose the ppm ports for networking here anymore?

Reply:
This docker-compose file is being used by the `do_kafka_test.sh` script so I'm not sure if exposing the ports is needed there. I'm pretty sure this module doesn't allow any traffic into it through ports like that so it should be fine, but something to check. We'll probably want to leave the name at "ppm" and pointing to the Dockerfile.standalone though.

Reply:
Thanks for the tip! The name has of the service been changed back to "ppm" and it is now pointing to Dockerfile.standalone again.

Reply:
The PPM still functions without those ports, so we're good 👍